gcsio/src/main/java/com/google/cloud/hadoop/gcsio/StorageStubProvider.java

```diff
@@ -66,7 +67,7 @@
   private final String userAgent;
   private final ExecutorService backgroundTasksThreadPool;
   private final List<ChannelAndRequestCounter> mediaChannelPool;
-  private final GrpcDecorator grpcDecorator;
+  @VisibleForTesting final GrpcDecorator grpcDecorator;
```

Comment:
How about we keep the instance variable `private` and add a `package-private` getter method ?

Reply:
Done.